odoo/addons/base/tests/ssl/openssl.conf

```diff
@@ -0,0 +1,85 @@
+# OpenSSL configuration file.
+# Largely inspired from https://jamielinux.com/docs/openssl-certificate-authority/appendix/root-configuration-file.html
+
+[ ca ]
+# `man ca`
+default_ca = CA_default
+
+[ CA_default ]
+# Directory and file locations.
+dir               = $ENV::OUTDIR
+certs             = $dir
+new_certs_dir     = $dir
+database          = $dir/index.txt
+serial            = $dir/serial
+RANDFILE          = $dir/.rand
+
+name_opt          = ca_default
+cert_opt          = ca_default
+default_days      = 1
+preserve          = no
+default_md        = sha256
+policy            = policy0
+
+[ policy0 ]
+# See the POLICY FORMAT section of `man ca`.
+countryName             = match
+localityName            = match
+commonName              = supplied
+emailAddress            = optional
+
+[ req ]
+# Options for the `req` tool (`man req`).
+default_bits        = 2048
+distinguished_name  = req_distinguished_name
+string_mask         = utf8only
+default_md          = sha256
+
+# Extension to add when the -x509 option is used.
+x509_extensions     = ca_cert
+
+[ req_distinguished_name ]
+# See <https://en.wikipedia.org/wiki/Certificate_signing_request>.
+countryName                     = Country Name (2 letter code)
+localityName                    = Locality Name
+commonName                      = Common Name
+emailAddress                    = Email Address
+
+[ ca_cert ]
+# Extensions for a typical CA (`man x509v3_config`).
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid:always,issuer
+basicConstraints = critical, CA:true
+keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+
+[ any_cert ]
+# Extensions for client certificates (`man x509v3_config`).
+basicConstraints = CA:FALSE
+nsCertType = server, client, email
+nsComment = "OpenSSL Generated Any Purpose Certificate"
+subjectKeyIdentifier = hash
+authorityKeyIdentifier = keyid,issuer
+keyUsage = critical, nonRepudiation, digitalSignature, keyEncipherment
```

Comment:
so maybe this can be removed too

Reply:
I've commented the code inside gencert.sh to not generate it anymore, and adapted the test to not load it either

Reply:
I'm not sure I'll remember those `openssl req` commands in a few months, I rather keep them commented for future-me